cves/CVE-2014-0050.yml

```diff
@@ -45,7 +45,10 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  This vulnerability allows for an infinite loop to occur, meaning the program is thus vulnerable for 
+  a DoS attack. The exploit would come in the form of sending an HTTP request with a Content-Type header
```

Comment:
Maybe add `"(Denial of Service)"` after DoS in case the reader doesn't know what it is.